Directory.Build.targets

```diff
@@ -66,6 +66,12 @@
          '$(IsBenchmarkProject)' == 'true' OR '$(IsSampleProject)' == 'true' OR '$(IsMicrobenchmarksProject)' == 'true') ">false</IsPackable>
   </PropertyGroup>
 
+  <!-- Properties for Package Validation -->
+  <!-- We only want to check for API changes in servicing branches -->
+  <PropertyGroup Condition=" '$(IsServicingBuild)' != 'true' ">
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

Comment:
Don't we need to set properties somewhere to inform Microsoft.DotNet.PackageValidation of the baseline package version once we _are_ in servicing❔

Reply:
correct we dont need this, baseline validation wont run by default